### DISCUSSION TITLE
Waiting for the build to show up in the build list - this may take a few minutes (check your email for processing issues if this continues) and I noticed that the version uploaded but not submitted to review

### DISCUSSION THREAD

**Discussion by Mahmoud-Bassuni** [fastlane/fastlane#19600] [Q&A]
### New Issue Checklist

- [ ] Updated fastlane to the latest version
- [ ] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [ ] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [ ] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

##### Complete output when running fastlane, including the stack trace and command used

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

### Environment

<details>
  <pre> [REPLACE THIS WITH YOUR INFORMATION] </pre>
</details>

[a comment by an automated account is not shown]

**Comment by crazymanish** [comment]
we need more details/logs or lane details.
also, this seems more like help or question so converting it into a discussion so the Fastlane community can help much faster after having more details.
